src/tox_current_env/hooks.py

```diff
@@ -38,6 +38,16 @@ def tox_addoption(parser):
         help="Don't run tests, only print the dependencies to the given file "
             + "(use `-` for stdout)",
     )
+    parser.add_argument(
+        "--print-extras-to",
+        "--print-extras-to-file",
```

Comment:
Consider removing `--print-extras-to-file` – there's no backwards compatibility here to keep.

Reply:
I'd keep it, it's not a maintenance burden and it makes things nicely similar.